File: C++/1848.cpp
#include <iostream>

using namespace std;

int calculate(string binary){
    if(binary == "---"){
        return 0;
    }else if(binary == "--*"){
        return 1;
    }else if(binary == "-*-"){
        return 2;
    }else if(binary == "-**"){
        return 3;
    }else if(binary == "*--"){
        return 4;
    }else if(binary == "*-*"){
        return 5;
    }else if(binary == "**-"){
        return 6;
    }else if(binary == "***"){
        return 7;
    }
}

int main() {
    string input;
    int num = 0;
    while(getline(cin, input)){
        if(input != "caw caw"){
            num += calculate(input);
        }else{
            cout << num << endl;
            num = 0;
        }
    }
    return 0;
}

File: Python/1848.py
count = 0

while True:
  try:
    linha = input()
    if linha == "--*":
        count += 1
    elif linha == "-*-":
        count += 2
    elif linha == "-**":
        count += 3
    elif linha == "*--":
        count += 4
    elif linha == "*-*":
        count += 5
    elif linha == "**-":
        count += 6
    elif linha == "***":
        count += 7
    elif linha == "caw caw":
        print(count)
        count = 0
  except EOFError:
    break
